tests/Fixtures/inc/Engine/Cache/WPCache/addWpCacheStatusTest.php

```diff
@@ -2,7 +2,24 @@
 
 return [
     'test_data' => [
+        'testShouldBailOutWhenSetCacheConstFilterFalse' => [
+            'config' => [
+				'set_filter_to_false'   => false,
```

Comment:
```suggestion
				'filter_constant_value'   => false,
```

The previous name is confusing to me, when reading it I thought we wanted to not set the filter to false, when it's the opposite.